WarehouseManagement.Core/Contracts/IEntryService.cs

```diff
@@ -9,7 +9,7 @@ public interface IEntryService
     Task<IEnumerable<EntryDto>> GetAllAsync(EntryStatuses[]? statuses);
     Task<IEnumerable<EntryDto>> GetAllByZoneAsync(int zoneId, EntryStatuses[]? statuses);
     Task<IEnumerable<EntryDto>> GetAllWithDeletedAsync(int? zoneId, EntryStatuses[]? statuses);
-    Task CreateAsync(EntryFormDto model, string userId);
+    Task CreateAsync(ICollection<EntryFormDto> model, string userId);
```

Comment:
What is the reason to receive a collection instead of a single entry DTO?

Reply:
I spoke with Kris about this. The project requirements document states that entries will be created when a new delivery is added. This means that multiple entries can be generated from a single delivery.

Reply:
Sure, but wouldn't it be better to simply make a foreach and then in its body to call CreateAsync for each entry?

Reply:
It might makes sense to be a different method too, no?

Reply:
It might not be necessary to change the method to accept a collection. If we need to create multiple items, we can simply call the method multiple times within a loop. This keeps the method signature simpler and focused on creating a single item.

Reply:
Guys this way we can still create one item. It is simple, just collection with single item will be passed and there you go. I would we have separate method for the same functionality.